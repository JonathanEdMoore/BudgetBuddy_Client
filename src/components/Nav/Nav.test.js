import React from 'react'
import ReactDOM from 'react-dom'
import Nav from './Nav'
import { BrowserRouter } from 'react-router-dom'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe('Nav smoke test', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<BrowserRouter><Nav /></BrowserRouter>, div)
    ReactDOM.unmountComponentAtNode(div)
  })
})

describe('Nav snapshot test', () => {
  it('renders the UI as expected', () => {
    const wrapper = shallow(<Nav />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})