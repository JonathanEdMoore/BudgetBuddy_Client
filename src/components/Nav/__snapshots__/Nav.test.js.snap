// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav snapshot test renders the UI as expected 1`] = `
<div>
  <div>
    <nav
      role="navigation"
    >
      <Link
        to="/signup"
      >
        <button>
          Sign Up
        </button>
      </Link>
      <Link
        to="/login"
      >
        <button>
          Log In
        </button>
      </Link>
    </nav>
  </div>
</div>
`;
